### ISSUE TITLE
Should catch missing <tr> tags

### ISSUE THREAD

**Issue by petdance** [petdance/html-lint#7]
Reported by project member petdance, Jun 2, 2008

HTML::Lint doesn't seem to pick up the following as a problem:

```
<table>
<td>
. . .
</td>
</table>
```

Note the missing `<tr></tr>`
